sections/design/prototypes.tex

```diff
@@ -0,0 +1,77 @@
+ The student then clicks the desired date and the popup window will show. 
+ Here they can specify the subject they wish to be tought and the timeslot as well as additional information. 
+ They can only choose one of tutors available subjects, and the timeslot should comply with the tutor's availability. 
+ The tutor also needs to choose what time they are available prior to this.
+ When the student makes the appointment they will be sent to a payment page, and a request will be sent to the tutor which they can either accept or deny. 
+
+ \begin{figure}[H]
+    \includegraphics[width=\linewidth]{/prototypes/view-information-on-tutor.png}
+     \caption{The page where other users can view information about a specific tutor}
+     \label{fig:view-information-on-tutor}
+ \end{figure}
+This is the page where other users can go and view information about another tutor. 
+A lot of information should be available here so that a user can know if this tutor can be helpful to them.
+The page will show basic information on the left side and also show the tutors rating. 
+A user can click \textit{see reviews} to see what other users wrote about the tutor. 
+The user viewing the tutor's page can also leave a review of the tutor or report the tutor if they had a bad experience. 
+The tutor can display some self written information about them in the middle that others can read. The middle also shows which subjects that the tutor teaches. 
+A tutor also has some material connected to their page if they have uploaded any. 
+They can make this available to others however they wish, but can also lock it behind payment or only make it available if a student books an appointment with the tutor.
+Users can click the \textit{View all material} button to go to the view material page where all the files and links can be found. 
+Finally the user can wish to make an appointment with the tutor. 
+By clicking the \textit{Make appointment} button the user is redirected to the tutors calendar where they can find a suitable date for the appointment. 
+
+
+ \begin{figure}[H]
+    \includegraphics[width=\linewidth]{/prototypes/view-tutor-material.png}
+     \caption{The page that shows a tutors available material}
+     \label{fig:view-tutor-material}
+ \end{figure}
+This page shows a specific tutors uploaded material. The material is divided into which subject the it belongs to make it easier for other users to find the relevant material. The material can be any kind of file or it can be links to useful pages such as some reading material or perhaps a youtube video. When a specific user goes to a tutor's material page, only the material that is accessible to them should be made available. All other files should show a lock on top of the file or link to indicate that this item is not available. The user can then hover over this item and it should then tell the user how it can be unlocked. 
```

Comment:
```suggestion
This page shows a specific tutor's uploaded material. 
The material is divided into which subject it belongs to, in order to make it easier for other users to find the relevant material. 
The material can be any kind of file or it can be links to useful pages such as some reading material or perhaps a youtube video. 
When a specific user goes to a tutor's material page, only the material that is accessible to them should be made available. 
All other files should show a lock on top of the file or link to indicate that this item is not available. 
The user can then hover over this item and it should then tell the user how it can be unlocked. 
```